src/Transport/Config/AzureServiceBusTransportInfrastructure.cs

```diff
@@ -72,10 +72,13 @@ void InitializeIfNecessary()
             topologyOperator = new TopologyOperator(messageReceiverLifeCycleManager, new BrokeredMessagesToIncomingMessagesConverter(Settings, new DefaultConnectionStringToNamespaceAliasMapper(Settings)), Settings);
         }
 
-        public override Task Start()
+        public override async Task Start()
         {
             InitializeIfNecessary();
-            return topologyManager.Initialize();
+
+            await topologyManager.Initialize().ConfigureAwait(false);
+            var resourcesToCreate = topologyManager.DetermineTopicsToCreate(Settings.LocalAddress());
```

Comment:
Nitpicking: topics are resources, but here the variable should probably be called `topicsToCreate`

Reply:
The method returns `TopologySectionInternal` so from that perspective the generic version is `resourcesToCreate` or `sectionToCreate` but I'm guessing being explicit does not hurt. I'll change it